src/main/java/com/hazelcast/web/WebFilterConfig.java

```diff
@@ -37,30 +37,135 @@
  * Contains all configuration parameters for Hazelcast session replication
  */
 public final class WebFilterConfig {
-    private static final ILogger LOGGER = Logger.getLogger(WebFilterConfig.class);
+    /**
+     * <p>Configuration key: <code>client-config-location</code></p>
+     * Location of the client's configuration. It can be specified as a servlet resource, classpath resource or as a URL.
```

Comment:
Lines could be limited to 80 or 90 or some certain length. Otherwise it looks messy when some of them have 120 where some others have 20 characters length.

Reply:
yes. got your point. will check accordingly.